poodle/src/styles/Mypage/ProgressBar/index.ts

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
+import { finishIcon, notFinishIcon } from '@/assets/Mypage';
+import { Link } from 'react-router-dom';
+
+export const ProgressBarWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 70px;
+`;
+
+export const ProgressBox = styled.div`
+    width: 980px;
+    height: 126px;
+    display: flex;
+`;
+
+export const ProgressItemBox = styled.div`
+    width: 91px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`;
+
+export const ProgressImageLink = styled(Link)<{ isfinished: string }>`
+    text-decoration: none;
+    width: 80px;
+    height: 80px;
+    background-size: contain;
+    margin-bottom: 24px;
+    background-image: url(${({ isfinished }) => isfinished === 'true' ? finishIcon : notFinishIcon});
+`;
+
+export const ProgressTitleLink = styled(Link)`
+    text-decoration: none;
+    width: 100px;
+    height: 22px;
+    font-family: NanumSquareR;
+    font-size: 20px;
+    line-height: 1.05;
+    text-align: center;
+    color: #afafaf;
+    letter-spacing: -2px;
+    `;
+    
+    export const Line = styled.hr`
```

Comment:
```suggestion
export const Line = styled.hr`
```
Prettier 사용하시면 이런 이슈 줄일 수 있습니다.

Reply:
오옹 prettier 맨날 적용이 안돼서 안썼는데 이번에 찾아보고 다시 해보니까 잘 작동하네요. 감사합니다.